repos/fdbt-site/src/pages/api/exports.ts

```diff
@@ -74,7 +74,11 @@ export const filterOutProductsWithNoActiveServices = async (
 ): Promise<DbProduct[]> => {
     const lineIdsToKeep: string[] = [];
 
-    const allBodsServices = await getBodsServicesByNoc(noc);
+    const services = await getAllServicesByNocCode(noc);
+    const hasBodsServices = services.some((service) => service.dataSource && service.dataSource === 'bods');
+    const tndsServices = services.filter((service) => service.dataSource && service.dataSource === 'tnds');
+    const dataSource = !hasBodsServices && tndsServices.length > 0 ? 'tnds' : 'bods';
+    const allBodsServices = await getBodsOrTndsServicesByNoc(noc, dataSource);
```

Comment:
cant we just decide on the datasource from the presence of the MULTI_MODAL_ATTRIBUTE ?